bundle/Controller/ContentEditController.php

```diff
@@ -69,7 +73,7 @@ public function __construct(
      * @param int $parentLocationId Location the content should be a child of
      * @param \Symfony\Component\HttpFoundation\Request $request
      *
-     * @return \Symfony\Component\HttpFoundation\Response
+     * @return \EzSystems\RepositoryForms\Content\View\ContentEditView|\Symfony\Component\HttpFoundation\Response
```

Comment:
Note: I'm not a huge fan of returning multiple unrelated object types. Of course refactoring everything now does not make sense, but please be aware of technical debt it creates when in some distant future we'll decide to use strict PHP7 :)

Reply:
Sure. Unfortunately we can't avoid doing so. `Response` objects doesn't allow any extensibility because they come with rendered HTML.